common/persistence/src/main/java/app/coronawarn/server/common/persistence/service/common/ValidDiagnosisKeyFilter.java

```diff
@@ -30,7 +51,10 @@
     return validDiagnosisKeys;
   }
 
-  private static boolean isDiagnosisKeyValid(DiagnosisKey diagnosisKey) {
+  /**
+   * Returns true if the given diagnosis key has passed the default entity validation.
+   */
+  public boolean isDiagnosisKeyValid(DiagnosisKey diagnosisKey) {
```

Comment:
Why not `static` any more?

Reply:
I guess it was just a preference of mine to include this method in the object's public interface. But it could just as well serve as a static method